### DISCUSSION TITLE
Master detail table

### DISCUSSION THREAD

**Discussion by maxs-rose** [taiga-family/taiga-ui#1238] [Q&A] [answered]
With the current system is it possible to create master detail table or is it just simply grouping rows that are available currently?

**Comment by waterplea** [comment, marked as answer]
Can you show some examples of what you're trying to achieve?

**Reply by maxs-rose** [reply]
What I would like to do is something similar to this https://www.primefaces.org/primeng/showcase/#/table/rowexpansion but I can't seem to figure out how 😅.

**Reply by waterplea** [reply]
If using current multiple tbody option doesn't work for you, maybe you can make accordion with separate tables?
